packages/devtools_app/lib/src/screens/memory/shared/heap/heap.dart

```diff
@@ -136,17 +125,40 @@ class SizeOfClassSet {
       shallowSize == 0 && retainedSize == 0 && instanceCount == 0;
 
   void countInstance(AdaptedHeapObject object) {
-    assert(!_isSealed);
+    assert(!isSealed);
     retainedSize += object.retainedSize!;
     shallowSize += object.shallowSize;
     instanceCount++;
   }
 
-  /// Mark the object as deeply immutable.
-  ///
-  /// There is no strong protection from mutation, just some asserts.
-  void seal() => _isSealed = true;
+  void uncountInstance(AdaptedHeapObject object) {
+    assert(!isSealed);
+    retainedSize -= object.retainedSize!;
+    shallowSize -= object.shallowSize;
+    instanceCount--;
+  }
+}
+
+/// Size of set of instances.
```

Comment:
this dart doc is the same as `ObjectSetStats`. Can you add some more details to differentiate the two?